NOTICE.txt

```diff
@@ -1494,6 +1494,50 @@ Files covered:
 spyder/images/winpython.svg
 
 
+-------------------------------------------------------------------------------
+
+
+
+OOjs UI MediaWiki lib
+---------------------
+
+Copyright (c) 2011-2018 OOUI Team and other contributors.
+
+
+Principal Authors:
```

Comment:
Instead of pasting all the authors here, to keep the format clean and consistent you can just paste this block (with the ``(major contributors, alphabetically)`` so as to be verbatim to the source) one blank line below the ``Licensed under the terms of the MIT license`` line, and use the simpler form for the authors:

```
Authors: OOUI Team and other contributors
https://phabricator.wikimedia.org/diffusion/GOJU/browse/master/AUTHORS.txt
Source: https://phabricator.wikimedia.org/foobar...
Download: https://commons.wikimedia.org/foobar...
```

That follows what I did in the other cases like this, so as to not overcomplicate things :)